inferentia/README.md

```diff
@@ -156,6 +158,24 @@ loaded on cores 2-3. To best engage inferentia device, try setting
 the number of neuron cores to be a proper multiple of the instance
 count.
 
+#### Batching with Pytorch
+
+There are three flags related to batched inputs:

```

Comment:
`gen_triton_model.py`  supports following three options for configuring (triton's dynamic batching)\<link to the triton dynamic batching\>:

1. --preferred_batch_size: <description with link to triton's docs>
2. --max_queue_delay_microseconds: <desc with link to triton's docs>
3. --disable_batch_requests_to_neuron: \<desc\>